lib/python/gladevcp/tooledit_gtk.glade

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <requires lib="gtk+" version="2.16"/>
+  <requires lib="gtk+" version="2.18"/>
```

Comment:
Jim, do you know why this changed and what affects it has?  Our glade files have a variety of these versions but none of them currently say 2.18.  They are mostly 2.16, but stepconf has some 3.2 and 3.4.